src/gui/MainWindow.cpp

```diff
@@ -104,6 +104,7 @@ MainWindow::MainWindow()
     #endif
 
     setWindowIcon(filePath()->applicationIcon());
+    m_ui->globalMessageWidget->setHidden(true);
```

Comment:
Looks like most of the time the MessageWidget instances are hidden when initialized. The `setHidden(true)` could be moved in the MessageWidget's constructor (since there is no .ui file for the widget).